pynucastro/rates/tabular_rate.py

```diff
@@ -33,8 +36,21 @@ class TableIndex(Enum):
     ('temp', numba.float64[:])
 ])
 class TableInterpolator:
-    """A simple class that holds a pointer to the table data and
-    methods that allow us to interpolate a variable"""
+    """A class that holds a pointer to the table data and
+    methods that allow us to interpolate a variable
+
+    Parameters
+    ----------
+    table_rhoy_lines : numpy.ndarray
```

Comment:
I think `table_rhoy_lines` and `table_temp_lines` are indices (int) for extracting rhoy and temp 1d array from table data.

Reply:
ah, indeed.

Reply:
fixed